docs/source/examples/pec-tutorial.myst

```diff
@@ -0,0 +1,541 @@
+---
+jupytext:
+  text_representation:
+    extension: .myst
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.10.2
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+# Probabilistic Error Cancellation (PEC) with Mitiq
+
++++
+
+This is set-by-sep tutorial on how to use the Mitiq toolchain for implementing **probabilistc error cancellation (PEC)** [[1-3]](#References).
+We use the [Cirq](https://github.com/quantumlib/cirq) library, but other frontends could be used in a similar way. 
+
+If you are only interested in **applying** PEC with Mitiq, the quick start is given at
+[this link](https://mitiq.readthedocs.io/en/stable/guide/guide-getting-started.html#probabilistic-error-cancellation) is probably a good start.
```

Comment:
```suggestion
If you are only interested in **applying** PEC with Mitiq, see the quick start
[here](https://mitiq.readthedocs.io/en/stable/guide/guide-getting-started.html#probabilistic-error-cancellation).
```

Reply:
Just rephrasing to sound more confident in our documentation: "Probably a good start"... I hope it's a good start!